pkg/token/daemon.go

```diff
@@ -172,7 +240,8 @@ func (d *daemon) writeFiles() error {
 	}
 
 	w := util.NewWriter()
-	d.accessTokenCache.Range(func(k CacheKey, t Token) error {
+
+	err := d.accessTokenCache.Range(func(k CacheKey, t Token) error {
```

Comment:
the current write file logic is all or nothing (NOT best effort), is it expected?